rebac-admin-backend/Makefile

```diff
@@ -22,5 +22,9 @@ mocks:
 
 # Run tests
 test: mocks
```

Comment:
I'd rather have two Makefile targets, one for coverage and another for just running the tests.

Reply:
I'll add a target that only does testing for local development purposes.
For the CI, I'd leave the coverage in if you agree

Reply:
That's the point. Thanks.